Core/include/Acts/Geometry/TrackingGeometry.hpp

```diff
@@ -48,10 +53,11 @@ class TrackingGeometry {
   ///        surface or volume based material to the TrackingVolume
   /// @param hook Identifier hook to be applied to surfaces
   /// @param logger instance of a logger (defaulting to the "silent" one)
+  /// @param close If true, run the Gen1 geometry closure
   TrackingGeometry(const std::shared_ptr<TrackingVolume>& highestVolume,
                    const IMaterialDecorator* materialDecorator = nullptr,
                    const GeometryIdentifierHook& hook = {},
-                   const Logger& logger = getDummyLogger());
+                   const Logger& logger = getDummyLogger(), bool close = true);
```

Comment:
potentially we could have a second constructor to keep the logger as a last argument. but I don't think it is a strong requirement

Reply:
As discussed, this has other drawbacks. We'll stick to this one until next breaking release.